torchvision/datasets/vision.py

```diff
@@ -5,6 +5,18 @@
 
 
 class VisionDataset(data.Dataset):
+    """
+    Base Class For making datasets which are compatible with torchvision.
+    It is necessary to override the ``__getitem__`` and ``__len__`` method.
+
+    Args:
+        root (string): Root directory of dataset.
+        transforms (callable, optional): A function/transform that takes in a PIL image
```

Comment:
Just saw that. What you describe here is the `transform` (note the missing `s` at the end). `transforms` needs to take an image and the label and return the transformed versions of both.